core/silentzap/levels_standard.go

```diff
@@ -0,0 +1,8 @@
+//go:build !tracelog
+
+package silentzap
+
+// allowedLevels returns allowed levels for the standard logger.
+func allowedLevels() string {
+	return `*"Debug" | "Info" | "Warn" | "Error" | "DPanic" | "Panic" | "Fatal"`
```

Comment:
Is there a reason we went for a function? Would prefer a const instead..

Reply:
Perfectly reasonable request. Making the change.